internal/dependency-track/GenerateBOMFile.sh

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env bash
+
+# Copyright (c) 2019, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+################################################################
+##   This script will generate a BOM file for given product   ##
+################################################################
+
+# ------------------------------------
+# Download source code
+# ------------------------------------
+
+read -p "Enter git source code download url to create BOM File : " download_url
+
+mkdir $HOME/workingDir
```

Comment:
shall we do this in `/tmp` and then delete the folder at the end? `/tmp/bom-generator`

Reply:
Done